package.json

```diff
@@ -47,7 +47,8 @@
     "chromatic:s2": "NODE_ENV=production CHROMATIC=1 chromatic --buildScriptName='build:chromatic-s2' --project-token=$RAINBOW_CHROMATIC_PROJECT_TOKEN",
     "merge:css": "babel-node --presets @babel/env ./scripts/merge-spectrum-css.js",
     "release": "lerna publish from-package --yes",
-    "publish:nightly": "lerna publish -y --canary --preid nightly --dist-tag=nightly --exact --force-publish='*' --no-push --no-verify-access",
+    "version:nightly": "yarn workspaces foreach --all --no-private -t version -d 3.0.0-nightly-$(git rev-parse --short HEAD)-$(date +'%y%m%d') && yarn version apply --all",
+    "publish:nightly": "yarn workspaces foreach --all --no-private -t npm publish --tag nightly",
```

Comment:
do we need the `--access public` flag here as per https://yarnpkg.com/cli/npm/publish#details?

Reply:
good catch, yes, probably need that

Reply:
I've been testing the publish like so https://github.com/adobe/react-spectrum/pull/6975#issuecomment-2316668208

But might be good to add some permissions to verdaccio